08_lisp.md

```diff
@@ -227,11 +254,12 @@ Similarly, we want `cdr` and [`rplacd`][set-car].
       return node(x).next;
     }
 
+[rplacd]: http://clhs.lisp.se/Body/f_rplaca.htm
+
 ### Free
 
 Now let's write `free`.
-We can make it somewhat more useful by returning something other than void.
-Return the next, otherwise the user will have to save it before freeing.
+This operation appends a pair to list making it available for reuse in another allocation.
```

Comment:
I'm not sure what's meant by "appends a pair to list". Could you say something about free "returning a list node to the pool"?